src/hunspell/affentry.cxx

```diff
@@ -217,7 +217,7 @@ struct hentry* PfxEntry::checkword(const char* word,
   // and if there are enough chars in root word and added back strip chars
   // to meet the number of characters conditions, then test it
 
-  int tmpl = len - appnd.size(); // length of tmpword
+  int tmpl = len - int(appnd.size()); // length of tmpword
```

Comment:
This is IMO a quite confusing/ambiguous way to write a cast.

Did you consider C++ style casts like static_cast? See http://cppblogs.blogspot.com/2013/08/c-style-casts.html for some examples.